### PR TITLE
profile title changes, css fixes

### PR DESCRIPTION
**Pull request by achen2401**
patient profile title changes, address this story: 
https://www.pivotaltracker.com/story/show/150118632
for patient profile title:
- use name if available, otherwise use email
- css fixes
- fix email saving (for no email)

### CODE REVIEW

File: portal/templates/profile.html [hunk 1]
```diff
@@ -22,19 +22,37 @@
...
+                          &nbsp;{{user.last_name}}
+                        {%- elif user.email -%}
+                          &nbsp;{{ user.email }}
+                        {%- else -%}
+                          &nbsp;#{{ user.id}}
+                        {%-endif-%}
+                    {%- else -%}
+                        {%- if current_user.id != user.id -%}
+                            {%- if user.email -%}
+                            {{ _("Profile for") }} {{ user.email }}
```
Comment (ivan-c) [inline portal/templates/profile.html]: Could you move the email/id into the translate block? Probably not strictly necessary but good practice :)
```
{% trans user_email=user.email %}Profile for {{user_email}}{% endtrans %}
```

Comment (achen2401) [inline portal/templates/profile.html]: @ivan-c 
Ivan, let me know if you see any problem.  I added trans statement where applicable in the profile title.

**Comment by ivan-c** [comment]
Sorry I didn't get a chance to talk to you about your new commit- should be good to merge now :+1:

**Comment by achen2401** [comment]
@ivan-c 
Ivan, ignorance on my part but how come the user's name doesn't need to be enclosed in trans tag?  ( it was originally assigned to a variable in the trans tag).  I saw your change, that is why I ask.  Will it be translated correctly?

**Comment by achen2401** [comment]
This should be ready for merging.  @ivan-c Ivan, please ignore my previous comment.  I did chat with Paul about it.

**Pull request merged**